Makefile

```diff
@@ -0,0 +1,339 @@
+	mv $(shell pwd)/bin/helm $(HELM) && chmod +x $(HELM) ;\
+	}
+else
+HELM = $(shell which $(HELM))
+endif
+endif
+
+# Download operator-sdk binary if necessary
+OPERATOR_SDK_RELEASE = v1.16.0
+OPERATOR_SDK = $(shell pwd)/bin/operator-sdk-$(OPERATOR_SDK_RELEASE)
+OPERATOR_SDK_DL_URL = https://github.com/operator-framework/operator-sdk/releases/download/$(OPERATOR_SDK_RELEASE)/operator-sdk_$(OS)_$(ARCH)
+operator-sdk:
+ifeq (,$(wildcard $(OPERATOR_SDK)))
+ifeq (,$(shell which $(OPERATOR_SDK) 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(shell pwd)/bin ;\
+	curl -sL -o $(OPERATOR_SDK) $(OPERATOR_SDK_DL_URL) ;\
+	chmod +x $(OPERATOR_SDK) ;\
+	}
+else
+OPERATOR_SDK = $(shell which $(OPERATOR_SDK))
+endif
+endif
+
+# Download kind locally if necessary
+KIND_RELEASE = v0.11.1
+KIND = $(shell pwd)/bin/kind-$(KIND_RELEASE) 2>/dev/null
+KIND_DL_URL = https://github.com/kubernetes-sigs/kind/releases/download/$(KIND_RELEASE)/kind-$(OS)-$(ARCH)
+kind:
+ifeq (,$(wildcard $(KIND)))
+ifeq (,$(shell which $(KIND) 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(shell pwd)/bin ;\
+	curl -sL -o $(KIND) $(KIND_DL_URL) ;\
+	chmod +x $(KIND) ;\
+	}
+else
+KIND = $(shell which $(KIND))
+endif
+endif
+
+# Download kuttl locally if necessary for e2e tests
+KUTTL_RELEASE = 0.9.0
+KUTTL = $(shell pwd)/bin/kuttl-v$(KUTTL_RELEASE)
+KUTTL_DL_URL = https://github.com/kudobuilder/kuttl/releases/download/v$(KUTTL_RELEASE)/kubectl-kuttl_$(KUTTL_RELEASE)_$(OS)_x86_64
+kuttl:
+ifeq (,$(wildcard $(KUTTL)))
+ifeq (,$(shell which $(KUTTL) 2>/dev/null))
+	@{ \
+	set -e ;\
+	mkdir -p $(shell pwd)/bin ;\
+	curl -sL -o $(KUTTL) $(KUTTL_DL_URL) ;\
+	chmod +x $(KUTTL) ;\
+	}
+else
+KUTTL = $(shell which $(KUTTL))
+endif
+endif
+
+####################################################
+#### Custom Targets clones original helm chart #####
+####################################################
+##@ Download Helm Chart
+
+HELM_CHARTS_PATH="$(PWD)/helm-charts"
+HELM_CHART_NAME="cert-name"
+HELM_CHART_REPO="https://charts.jetstack.io"
+download-helm-chart: helm ## Download original helm chart into operator directory helm-charts/
+	@$(HELM) repo add jetstack $(HELM_CHART_REPO) >/dev/null \
+		|| { echo "Unable to add $(HELM_CHART_REPO) reposirtory"; exit 1; }
+	@test -d $(HELM_CHARTS_PATH)/cert-manager \
+		|| { mkdir -p $(HELM_CHARTS_PATH)/cert-manager; } \
+		&& { find $(HELM_CHARTS_PATH)/cert-manager -delete; }
+	@$(HELM) pull \
+		--untar --untardir ${HELM_CHARTS_PATH} \
+		--version=$(VERSION) jetstack/cert-manager
+	@sed -i "s|installCRDs: [^ ]*|installCRDs: false|g" \
+		$(HELM_CHARTS_PATH)/cert-manager/values.yaml
+
+TMP_FOLDER="$(PWD)/tmp"
+
+render-cdrs-from-helm-chart: helm download-helm-chart ## Generates the CRD definitions from the existing helm chart
+	@test ! -d $(TMP_FOLDER)/render || find $(TMP_FOLDER)/render -delete;
+	@$(HELM) template --set installCRDs=true --include-crds $(HELM_CHARTS_PATH)/cert-manager \
```

Comment:
Why set `installCRDs=true` when in the previous  **download-helm-chart** target you are already doing a safe **sed** with the correct value `false`?

Reply:
This step is only used to generate the CRDs, in a temporary folder, that's why that flag is required here but not in the download step.